lib/DataRepeater.js

```diff
@@ -464,13 +476,20 @@ var DataRepeater = module.exports = kind(
 		// access members of our dataset consistently, regardless
 		// of whether our data is in an array or a Collection
 		if (c && !c.at) {
-			c.at = at.bind(c);
+			Object.defineProperty(c, 'at', {value: at, enumerable: false});
 		}
 		if (p && p.removeListener) {
 			for (e in this._handlers) {
 				p.removeListener(e, this._handlers[e]);
 			}
 		}
+		if (c && this.selectionProperty) {
+			filterSelectionProperty = this.selectionProperty;
```

Comment:
Going to make a small tweak so that we don't need to maintain this var, and instead will pass the context to the `filter` method.